app/controllers/admin/territories/motifs_controller.rb

```diff
@@ -0,0 +1,49 @@
+class Admin::Territories::MotifsController < Admin::Territories::BaseController
+  def index
+    @organisations = current_territory.organisations
+    @services = current_territory.services.reject(&:secretariat?)
```

Comment:
On retrouve exactement ce code `current_territory.services.reject(&:secretariat?)` dans `app/views/admin/motifs/form/_configuration_principale.html.slim`, c'est à dire pour définir quels services sont disponibles dans le select lors de la création / édition d'un motif.

J'hésite à en faire une méthode de classe du genre `Service.for motifs_of(territory)`, mais je ne suis pas sûr de l'avantage que cette factorisation / abstraction apporte.